pkg/users/parsers.js

```diff
@@ -77,3 +77,19 @@
 export const etc_group_syntax = {
     parse: parse_group_content
 };
+
+function parse_shells_content(content) {
+    content = (content || "").trim();
+    if (!content) {
+        console.warn("Couldn't read /etc/shells");
+        return [];
+    }
+
+    const lines = content.split('\n');
+
+    return lines.filter(line => !line.contains("#") && !line.strim().isEmpty());
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18234-20230202-125519-871b841d-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)